### ISSUE TITLE
wrong calculated width for outlook comments (TDs) of two columns.

### ISSUE THREAD

**Issue by shauste** [mjmlio/mjml#584]
I use two columns – 40% 60%, the width of the container is 640px.

The comments for outlook (if mso | IE) shows a width of 640 for both td-elements, thats wrong.
If i add a mj-group around the columns it calculates the correct width of both columns (256px 384px).

https://mjml.io/try-it-live/Hk7eLdhcx

```xml
<mjml>
  <mj-head>
    <mj-attributes>
      <mj-all font-size="15" />
      <mj-all padding="0" />
    </mj-attributes>
  </mj-head>
  <mj-body>
    <mj-container width="640" background-color="#FFFFFF">
      <mj-section padding="0 25 25 25" background-color="red">
        <mj-column width="40%">
          <mj-image src="https://unsplash.it/270/190/?random" width="270" align="left" padding-right="25" padding-bottom="15" />
        </mj-column>
        <mj-column width="60%">
          <mj-text color="#FFF" padding-bottom="10">Nullam dictum felis eu pede mollis pretium. Integer tincidunt. Cras dapibus. Vivamus elementum semper nisi.</mj-text>
          <mj-button align="left" color="#FFFFFF" background-color="#000" color="#FFFFFF" border="1px solid #FFFFFF" border-radius="1px" font-weight="bold" padding-left="0" padding-right="0" inner-padding="8 15" href="#">MEHR…</mj-button>
        </mj-column>
      </mj-section>
    </mj-container>
  </mj-body>
</mjml>
```

  **Email clients the bug is seen on:**

Outlook all versions

**Comment by iRyusa** [comment]
Hi @shauste 

This should be fixed in MJML 3.3 beta 7, can you take a look at it ? Thanks !

**Comment by shauste** [comment]
Hi @iRyusa 

it works in the beta, thanks!
